go/teams/member_test.go

```diff
@@ -716,6 +716,37 @@ func TestOnlyOwnerLeaveThenUpgradeFriend(t *testing.T) {
 	}
 }
 
+func TestLeaveAsReader(t *testing.T) {
```

Comment:
does this test break on the broken build?

Reply:
Yes